app/config/parameters.yml.dist

```diff
@@ -14,7 +16,5 @@ parameters:
     mailer_user:       ~
     mailer_password:   ~
 
-    locale:            en
-
     # A secret key that's used to generate certain security-related tokens
     secret:            ThisTokenIsNotSoSecretChangeIt
```

Comment:
correct, then I would suggest to rephrase the comment to "Set parameters here that may be different on each deployment target of the app (development, staging, production)"